.github/workflows/e2e-tests-on-PR.yml

```diff
@@ -0,0 +1,33 @@
+name: PR->master
+
+on:
+  pull_request:
+    branches: [ master ]
+
+jobs:
+
+  cypress:
+    name: "Cypress"
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+
+    - name: launch the world
+      run: make install && docker-compose up -d
+
+    - name: run the tests
+      run: cd ./e2e-tests && npm install && npx cypress run
```

Comment:
Can we change `npx cypress run` to an npm script so we can run it locally with a single command - `npm run test:e2e` would keep it consistent